Bottom: 89b6fdb21835a9020a497d2b864f4a2ac8e4e614
Top:    bfc8629a7ae9afb987ba0e7b1c7fc0562950223f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:06:40 +0000

Refresh of adjust-coordinate-system

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 790b7d1..5437690 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -387,7 +387,7 @@ object3d_from_contours (const POLYAREA *contours,
 #ifdef REVERSED_PCB_CONTOURS
       face3d_set_normal (faces[npoints], 0., 0., -1.);
 #else
-      face3d_set_normal (faces[npoints], 0., 0., 1.); /* PCB bottom is at positive Z in this scheme */
+      face3d_set_normal (faces[npoints], 0., 0., 1.);
 #endif
       face3d_set_appearance (faces[npoints], top_bot_appearance);
       object3d_add_face (object, faces[npoints]);
@@ -396,7 +396,7 @@ object3d_from_contours (const POLYAREA *contours,
 #ifdef REVERSED_PCB_CONTOURS
       face3d_set_normal (faces[npoints + 1], 0., 0., 1.);
 #else
-      face3d_set_normal (faces[npoints + 1], 0., 0., -1.); /* PCB top is at negative Z in this scheme */
+      face3d_set_normal (faces[npoints + 1], 0., 0., -1.);
 #endif
       face3d_set_appearance (faces[npoints + 1], top_bot_appearance);
       object3d_add_face (object, faces[npoints + 1]);
